app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -30,68 +31,91 @@ const NavItem = ({
 
 const Navbar = () => {
   const [navMenuVisible, setNavMenuVisible] = useState(false);
+  const { t, i18n } = useTranslation();
 
   return (
     <S.Header>
       <a href="https://www.habitatmap.org/" aria-label="AirCasting page">
-        <S.AircastingLogo alt="Aircasting logo" src={logo} />
+        <S.AircastingLogo alt={t("navbar.altLogo")} src={logo} />
       </a>
       <S.Container>
-        <S.BuyCTA href={urls.airbeamBuy}>get airbeam</S.BuyCTA>
+        <S.BuyCTA href={urls.airbeamBuy}>
+          {t("navbar.sections.getAirBeam")}
```

Comment:
Maybe we can adjust, so the names are corresponding? Here I like `getAirbeam` better.

Reply:
I meant, please change the key in urls object, so it is consistent with the translation key. :)

Reply:
Just see how airbeam is written. ;) `getAirbeam`